change/@ni-nimble-angular-51bf35f5-d259-451f-990e-2c0574575308.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
After this PR completes and packages are published could you double check that
- the License section on [npmjs.com for packages like nimble-angular](https://www.npmjs.com/package/@ni/nimble-angular) gets updated
- the License Info section on [NimbleBlazor on nuget.org](https://www.nuget.org/packages/NimbleBlazor/) gets updated

Reply:
Will do.